### DISCUSSION TITLE
Modal sticky Title

### DISCUSSION THREAD

**Discussion by alibugrat** [saadeghi/daisyui#669] [Q&A] [answered]
Hi Guys,

I am not sure is this possible. I could not find any example. Can i make sticky modal title? I am using modal like this;
```
<label for="followingUsers-modal" class="modal cursor-pointer">
  <label class="modal-box relative">
     <div class="modal-title">
        <h4 class="font-medium leading-tight text-2xl mt-0 mb-2 text-blue-900">Following</h4>
     </div>
     <div class="modal-dialog-scrollable p-4">...</div>
  </label>
</label>
```
But even though i put scrollable in dialog. its make title scrollable too.

**Comment by saadeghi** [comment, marked as answer]
You can use `sticky top-0` for the title like this:  
https://play.tailwindcss.com/HTXySsmMuu
